cpu/atmega_common/periph/gpio.c

```diff
@@ -74,10 +74,11 @@ static inline uint16_t _port_addr(gpio_t pin)
     port_addr += GPIO_BASE_PORT_A;
     port_addr += GPIO_OFFSET_PIN_PORT;
 
+#if !defined(__AVR_ATmega328P__)
     if (port_num > PORT_G) {
         port_addr += GPIO_OFFSET_PORT_H;
     }
-
+#endif
```

Comment:
Sorry I think I don't understand what you mean... If I remove the #if !defined(__AVR_ATmega328P) it won't build exactly because PORT_G is not defined ...? 
@kYc0o can you clarify please ?